CONTRIBUTING.md

```diff
@@ -61,6 +61,7 @@ Supported filename extensions are:
 
 - ``.feature``: Signifying a new feature in Element Android or in the Matrix SDK.
 - ``.bugfix``: Signifying a bug fix.
+- ``.dev``: Signifying a change which is not ready to be user facing, typically a component of a larger feature.
 - ``.doc``: Signifying a documentation improvement.
 - ``.removal``: Signifying a deprecation or removal of public API. Can be used to notifying about API change in the Matrix SDK
```

Comment:
Maybe take the opportunity to create a `.sdk` extension, this one is not obvious at first sight.

Reply:
do we want to replace `.removal` with `.sdk`?

Reply:
`.removal` is available out of the box, so I decided to hack it and use it to log the SDK change (especially in the API). Maybe just add `.sdk` and start using it is enough. Also need to update https://github.com/vector-im/element-android/blob/main/towncrier.toml#L23

Reply:
if I understood correctly https://github.com/vector-im/element-android/pull/4953/commits/7259324b3ba1d871ba389a8239a5acc592e22110

if we want to keep the `removal` category is there a preferred section title?

Reply:
I think we will stop using `.removal`